services/CurrentTime/include/ble-service-current-time/CurrentTimeService.h

```diff
@@ -125,6 +125,13 @@ class CurrentTimeService {
 
         bool to_tm(struct tm * remote_time_tm);
 
+        uint16_t get_year() {
+            uint8_t *year_data  = (uint8_t *)&year;
+            uint16_t year_value = *year_data;
+            year_value |= *year_data << 8;
+            return year_value;
```

Comment:
You can just return `year` for now and leave a comment that the code should be modified when BE support is added.